src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -106,6 +108,18 @@ class AutomaticSpeechRecognitionPipeline(ChunkPipeline):
     The input can be either a raw waveform or a audio file. In case of the audio file, ffmpeg should be installed for
     to support multiple audio formats
 
+    Example:
+
+    ```python
+    >>> from transformers import pipeline
+
+    >>> transcriber = pipeline(model="openai/whisper-base")
+    >>> transcriber("https://huggingface.co/datasets/Narsil/asr_dummy/resolve/main/1.flac")
+    {'text': ' He hoped there would be stew for dinner, turnips and carrots and bruised potatoes and fat mutton pieces to be ladled out in thick, peppered flour fat and sauce.'}
+    ```
+
+    [Using pipelines in a webserver or with a dataset](../pipeline_tutorial)
```

Comment:
I think your phrasing is better, and that we should include how to use in a webserver or with a dataset, in the tutorial !

If you're OK since I used that link in all the follow-ups, I'm going to merge the PR as-is, merge all the subsequent ones and do a massive rename afterwards it's going to be easier for me than manually modifying all PRs.
And the new PRs I'm about the create I'll include your phrasing.

Reply:
Thanks! ❤️